### ISSUE TITLE
We need to improve the update commit message

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode-licensedb#27]
See https://github.com/nexB/scancode-licensedb/blob/4c45c3fa5fb2e21147148882f30d9e3a13d17f85/.github/workflows/daily-update.yml#L34

This should be using a DCO and there should be a link to the SCTK commit.

**Comment by AyanSinhaMahapatra** [comment]
Yes, I was confused what name/email to use for this signoff, what do you think?

**Comment by pombredanne** [comment]
whatever user is the one committing

**Comment by nilaygit-10721** [comment]
i make pr please check https://github.com/aboutcode-org/scancode-licensedb/pull/60
